tools/release_checks.rb

```diff
@@ -42,55 +41,34 @@ def check_tag
     end
   end
 
-  def check_jenkins_state(jenkins_url)
-    request = generate_request(jenkins_url)
-    json = JSON.parse(get_response(request))
-
-    actions = json["actions"].reject(&:empty?)
-    last_revision_tested?(get_last_revision, get_tested_revision(actions))
-
-    get_result(json["result"])
-  end
-
-  def get_response(request)
-    response = http.request(request)
-    if response.code != "200"
-      fail "Could not download Jenkins state information. Abort."
-    else
-      response.body
-    end
-  end
-
-  def generate_request(url)
+  def check_jenkins_state(url)
     uri = URI.parse(url)
+
     http = Net::HTTP.new(uri.host, uri.port)
     http.use_ssl = uri.scheme == "https"
-    Net::HTTP::Get.new(uri.request_uri)
-  end
+    request = Net::HTTP::Get.new(uri.request_uri)
+    response = http.request(request)
 
-  def get_result(response)
-    result = response
-    if result != "SUCCESS"
-      fail "Current HEAD does not build successfully in Jenkins. Abort."
-    end
-    result
-  end
+    if response.code == "200"
+      json = JSON.parse(response.body)
 
-  def get_tested_revision(actions)
-    tested_revision = actions.find do |e|
-      e.has_key?("lastBuiltRevision")
-    end["lastBuiltRevision"]["SHA1"]
-    tested_revision
-  end
+      actions = json["actions"].reject{ |h| h.empty? }
```

Comment:
Space missing to the left of {.<br>Pass `&:empty?` as an argument to `reject` instead of a block.